### PR TITLE
implement tests

### PR DESCRIPTION
**Pull request by bellatoris**

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #151** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/151" target="_blank">Graphite</a> 
    * **PR #152** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/152" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/152?utm_source=stack-comment).

**Review by Zeniuus — APPROVED** [review]

File: subprojects/bounded_context/accessibility/infra/src/integrationTest/kotlin/club/staircrusher/accesssibility/infra/adapter/in/controller/AccessibilityRankTest.kt [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            .accessibilityRank
+        assertEquals(1L, rank.rank)
+
+        val leaderboard2 = mvc
+            .sccRequest("/getAccessibilityLeaderboard", null)
+            .getResult(GetAccessibilityLeaderboardPost200Response::class)
+        val secondRanker = leaderboard.ranks.first { it.rank != 1L }
+        val secondRanker2 = leaderboard2.ranks.first { it.rank != 1L }
+        assertEquals(secondRanker.user, secondRanker2.user)
+        assertEquals(secondRanker.rank + 1, secondRanker2.rank)
```
[the remark this answers is not in the document]
Comment (bellatoris) [reply subprojects/bounded_context/accessibility/infra/src/integrationTest/kotlin/club/staircrusher/accesssibility/infra/adapter/in/controller/AccessibilityRankTest.kt:87]: 그럼 운이 억수로 좋은게 아닐까 싶습니다

**Comment by bellatoris** [comment]
@bellatoris queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/152).

**Comment by bellatoris** [comment]
[Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/152) rebased this pull request as part of a merge.

**Comment by bellatoris** [comment]
@bellatoris merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/152).

**Comment by bellatoris** [comment]
[Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/152) failed to merge this pull request for an unknown reason. Try your merge again, or report a bug if you see this consistently.

**Pull request merged**